### ISSUE TITLE
Accessibility: Podcast page expand has accessibility issues

### ISSUE THREAD

**Issue by pocketcasts** [Automattic/pocket-casts-ios#2]
### User report

"Expand Podcast” Button does not work, after double-tapping on it Nothing happens, podcast is not expanded so I can’t read what the podcast is about.

### Additional information

It appears this is related to the podcast header being a UITableViewCell with content that appears. Tried firing off all sorts of accessibility notifications but it didn't seem to help. This might need to be part of a reworking of the podcast page, since that's a complex page.
